src/ray/core_worker/store_provider/memory_store/memory_store.h

```diff
@@ -62,6 +71,15 @@ class CoreWorkerMemoryStore {
   bool Contains(const ObjectID &object_id);
 
  private:
+  /// Optional callback for putting objects into the plasma store.
+  std::function<void(const RayObject &, const ObjectID &)> store_in_plasma_;
```

Comment:
Is this actually optional?

Reply:
Yeah, it CHECK fail if not provided for real. The reason it's optional is to have a nice default construct interface for unit testing.